client/data/php-versions/index.jsx

```diff
@@ -27,7 +28,7 @@ export const getPHPVersions = () => {
 			// translators: PHP Version for a version switcher
 			label: sprintf( __( '%s (Deprecated)' ), '8.1' ),
 			value: '8.1',
-			disabled: false, // EOL 31st December, 2025
+			disabled: ! PHP81BrokenSites.includes( siteId ?? 0 ), // EOL 31st December, 2025
```

Comment:
If you set a default value for the param, you can simplify this:

```suggestion
			disabled: ! PHP81BrokenSites.includes( siteId ), // EOL 31st December, 2025
```